backend/db/models/session.js

```
@@ -85,6 +102,19 @@ class SessionModel {
     return session;
   }
 
+  async findByShelltenderSessionId(shelltenderSessionId) {
+    const session = await this.db.get(`
+      SELECT * FROM terminal_sessions 
+      WHERE shelltender_session_id = ?
+    `, [shelltenderSessionId]);
+
+    if (session) {
+      session = this._parseSession(session);
```

Biome (2.1.2)

[error] Can't assign session because it's a constant.

This is where the variable is defined as constant.

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)